### PR TITLE
UIIN-3063: ECS: Item affiliation cannot be changed when it is attached to a local order

### PR DESCRIPTION
**Pull request by alisher-epam**
## Purpose

[UIIN-3063](https://folio-org.atlassian.net/browse/UIIN-3063): ECS: Item affiliation cannot be changed when it is attached to a local order
## Approach

https://github.com/user-attachments/assets/b0195733-2a57-42a1-be3d-240426e05dc5

### CODE REVIEW

**Review by SerhiiNosko — APPROVED** [review]

**Comment by usavkov-epam** [comment]
Please fix sonar issue

File: src/ViewHoldingsRecord.js [hunk 1]
```diff
@@ -908,6 +961,7 @@ class ViewHoldingsRecord extends React.Component {
                     await switchAffiliation(stripes, tenantFrom, this.onClose);
                   }}
                   actionMenu={this.getPaneHeaderActionMenu}
+                  test="holdings-record-view-pane"
```
Comment (OleksandrHladchenko1) [inline src/ViewHoldingsRecord.js:964]: Is there any purpose of using `test` prop in `Pane` component?

**Review by mariia-aloshyna — APPROVED** [review]

File: src/Instance/MoveHoldingContext/MoveHoldingContext.js [hunk 2]
```diff
@@ -191,8 +198,9 @@ const MoveHoldingContext = ({
     const from = target.dataset.itemId;
     const isHolding = target.dataset.isHolding;
     const fromSelectedMap = selectedItemsMap[from] || {};
-    const selectedColumnHoldings = leftInstance.id === to ? rightHoldings : leftHoldings;
-    const holdingIds = selectedColumnHoldings.map(i => i.id);
+    const selectedInstanceHoldings = leftInstance.id === to ? rightHoldings : leftHoldings;
+    const selectedInstanceHoldingsHoldingIds = selectedInstanceHoldings.map(i => i.id);
```
Comment (usavkov-epam) [inline src/Instance/MoveHoldingContext/MoveHoldingContext.js:202]: Maybe just `selectedInstanceHoldingsIds`?

**Review by usavkov-epam — APPROVED** [review]

File: src/Instance/MoveHoldingContext/MoveHoldingContext.js (hunk 2 above)
Comment (alisher-epam) [reply src/Instance/MoveHoldingContext/MoveHoldingContext.js:202]: Good catch

**Pull request merged**